README.md

```diff
@@ -57,6 +58,9 @@ patterns of the GitHub appliance. We recommend allocating at least 5x the amount
 of storage allocated to the primary GitHub appliance for historical snapshots
 and growth over time.
 
+As Repository, Search, and Pages data is stored efficiently via hard links,
+backup snapshots must be written to a filesystem which supports hard links.
```

Comment:
Mentioning the specific data that uses hard links makes this fragile to future changes, so I'd leave the specifics out. A link to a good, concise description of hard links would help too.

How about:

> The backup utilities use [hard links]() to store data efficiently, so the backup filesystem used must support hard links.